requirements.txt

```diff
@@ -1,6 +1,5 @@
-tornado==4.5.3
+tornado==5.1.1
 pynsq
-futures==3.1.1
 mdict
 redis
-git+ssh://git@github.com/augurysys/auguryapi-py.git
+git+https://github.com/augurysys/auguryapi-py.git#egg=auguryapi
```

Comment:
why is that? the 'egg' need https URL?

Reply:
not sure, did not make this change, will try to revert

Reply:
seems to work